### ISSUE TITLE
Automate releases, make sure git tags are added

### ISSUE THREAD

**Issue by obo** [ELITR/SLTev#54]
We need to have a very simple way of figuring out the commit ID given the simple numbered version of a released SLTev. That's what tags are for. Please use git tags -- and please add manually the tag for the current version at least, so far no tag has been created.

To make sure we no longer forget to create tags, please write a 'make-release' script which will do all the necessary steps, including the creation of the tag.

I am assigning this to Mohammad but adding Matus, because Matus will know how to this releasing and tagging properly.

**Comment by mzilinec** [comment]
I would simply do it manually, each time there's a new release, increment version number in `setup.py`, tag the commit and push to PyPI. (But always for the same commit!) Also would be nice to use semantic versioning https://semver.org/

**Comment by obo** [comment]
I agree manual versioning should be sufficient, but it doesn't quite work. I have asked Mohammad to tag the released versions a while ago and it didn't help. I can understand that he keeps forgetting.

**Comment by mohammad2928** [comment]
Hi,
I did not forget and just waited for the stable version. 
I have created a simple script by the name "auto-tagging.py" and after each committed, we need to change the "__version__" variable in the SLTev/__init__.py module and call auto-tagging as follow:
python auto-tagging
